app/celery/nightly_tasks.py

```diff
@@ -216,16 +217,15 @@ def raise_alert_if_letter_notifications_still_sending():
 def get_letter_notifications_still_sending_when_they_shouldnt_be():
     today = datetime.utcnow().date()
 
-    # Do nothing on the weekend
-    if today.isoweekday() in {6, 7}:  # sat, sun
-        return 0, None
+    bank_holidays = BankHolidays()
```

Comment:
So we can change things in one place I’d use the definition from utils:

```suggestion
    bank_holidays = non_working_days_dvla
```

With `from notifications_utils.letter_timings import non_working_days_dvla`

Reply:
nice 💅

Reply:
Suggests I should probably be adding a `get_prev_dvla_working_day` to utils though. Love 2 PRs for the sake of 1 😓